src/models/spaces/organisms/organisms-space.ts

```diff
@@ -123,7 +123,15 @@ export const OrganismsSpaceModel = types
           organismsMouse = OrganismsMouseModel.create({ backpackMouse });
           self.organismsMice.push(organismsMouse);
         }
-        self.rows[rowIndex] = OrganismsRowModel.create({ organismsMouse });
+
+        // sync initial slider position
+        let proteinSliderStartPercent = 0;
```

Comment:
Now that I think about this, would it be better to store this value in the `organisms-state` model instead? It doesn't seem like there's a time when rows will have different values

Reply:
We're synching because I figured that that makes the most sense compared with the old model, but I don't know that it will always be like that. I could imagine at some point students might even select two different kinds of proteins, so then it wouldn't make sense to sync.

If we were always definitely synching, I think putting it there would make sense.